docker/.env

```
@@ -237,11 +237,15 @@ SMQ_USERS_ADMIN_LAST_NAME=admin
 SMQ_USERS_PASS_REGEX=^.{8,}$
 SMQ_USERS_ACCESS_TOKEN_DURATION=15m
 SMQ_USERS_REFRESH_TOKEN_DURATION=24h
-SMQ_TOKEN_RESET_ENDPOINT=/reset-request
 SMQ_USERS_HTTP_HOST=users
 SMQ_USERS_HTTP_PORT=9002
 SMQ_USERS_HTTP_SERVER_CERT=
 SMQ_USERS_HTTP_SERVER_KEY=
+SMQ_USERS_GRPC_HOST=users
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMQ_USERS_GRPC_HOST key should go before the SMQ_USERS_HTTP_HOST key

(UnorderedKey)